[ADD] accounting: new shared accounts feature documentation (#11769)
odoo/documentation: content/applications/finance/accounting/get_started/shared_accounts.rst

===============
Shared Accounts
===============

In Odoo, accounts can be shared between multiple companies. This feature is particularly useful for
businesses that operate under a **multi-company structure** and might benefit from a
**unified chart of accounts**.

Two ways are available to set up said feature: the :ref:`basic setup <setting_up_shared_accounts>`
or using the :ref:`merge tool <merge_tool>`.

> The merging tool allows existing multi-company environments to use the new shared accounts feature.

Reviewer: ```suggestion
The merge tool allows existing multi-company environments to use the new shared accounts feature.
```

You call it the "merge tool" above, so let's keep things homogeneous. If we prefer "merging tool", it's fine with me as well :)

This feature can also facilitate the :doc:`consolidation` of financial data across a multi-company
environment, providing a **global view** of the financial situation for a group with child
companies under a parent company.

.. _shared_accounts_benefits:

Benefits of Shared Accounts
===========================

1. Lighter load of accounts to handle
-------------------------------------

The main benefit of the shared accounts feature is to **reduce the amount of accounts** that have
to be handled by the main accountant of a multi-company environment.

.. example::
   In environments with two companies using the same country’s generic chart of accounts, each
   account used to appear twice. With the shared accounts feature, each account can now appear only
   once.

2. Simplified and Cleaner Reporting
-----------------------------------

The shared accounts feature **simplifies audit reports** while **keeping all data visible and
accessible**, making the accounts easier to understand and manage.

.. example::
   In the balance sheet report, shared accounts between a company and its child company appear in a

>    single group, differentiated by the company using each account.

Reviewer: I don't get what you mean by "differentiated by the company using each account" :thinking:

3. Macro and Micro Management
-----------------------------

The shared accounts feature allows accountants to manage accounts at the appropriate level for each
company. Main accountants can oversee the accounts of all child companies, clearly identifying each
account’s purpose, even if the codes differ across companies. Meanwhile, child companies'
accountants can manage their own accounts without worrying about their impact on other companies
using the same account.

.. example::

>    Even though the account is shared and exists as a single record in the database for all

Reviewer: Not sure this is really an example :thinking:  Plus, is this the right place to talk about "records ? Isn't it too technical ? :thinking:

   companies, its code can be modified in each company without affecting the others, granting more
   flexibility in account management.

.. _setting_up_shared_accounts:

Setting Up Shared Accounts
==========================

Enable Multi-Company
--------------------

> To enable account sharing and the :doc:`multi-company </applications/general/multi_company>`

Reviewer: ```suggestion
To enable account sharing and :doc:`multi-company </applications/general/multi_company>`
```

> feature, set up multiple companies in the database.

Reviewer: ```suggestion
features, set up multiple companies in the database.
```

a 6-digit code in the corresponding company :guilabel:`Code` column to share the account.

The account will then be included in the :doc:`consolidation` process. To make it available for
invoices in companies where a code has been configured, go to the :guilabel:`Accounting` tab and
add a second company to the list of :guilabel:`Companies` that can use this account.

.. _merge_tool:

Merge Tool
==========

Use cases
---------

The merging tool simplifies creating shared accounts and is particularly useful when a
**multi-company environment has been set up** with separate accounts that need to be merged.

Steps
-----

To use the merge tool, select all the companies with an account that needs to be merged in the
company selector in the top right corner of the screen.

.. image:: shared_accounts/shared_accounts_merge_tool_select_companies.png
   :alt: Selecting all companies that have accounts to be merged.

Then, go to :menuselection:`Accounting --> Configuration --> Chart of Accounts` and select the
accounts to merge. Click the :icon:`fa-cog` :guilabel:`Actions` menu and select :guilabel:`Merge
accounts`.

> In the :guilabel:`Merge accounts window, enable the :guilabel:`Group by name?` option if needed and

Reviewer: The second :guilabel: on this line is broken

click :guilabel:`Merge`.

The selected accounts are then merged into a single shared account, accessible by all the chosen

> companies, just as if the account had been created for one company and shared in the :ref:`basic
> setup <setting_up_shared_accounts>`.

Reviewer: ```suggestion
companies, just as if the account had been directly created to be shared.
```
Simpler IMO

.. _unmerge_tool:

Unmerge Tool
============

Use cases
---------

The unmerge tool simplifies stopping the sharing of specific accounts and allows **dissociating
accounts from certain companies** without losing their data.

Steps
-----

To unmerge accounts, select a company with a shared account in the top right corner of the screen.
Then, go to :menuselection:`Accounting --> Configuration --> Chart of Accounts` and select the
account to unmerge. Click the :icon:`fa-cog` :guilabel:`Actions` menu and select
:guilabel:`Unmerge accounts`.

> An :guilabel: Odoo Warning` confirmation pop-up window will appear, listing how the accounts will

Reviewer: Broken :guilabel : :fire:

   to have a single version of each of those accounts.

2. Simplified and Cleaner Reporting
-----------------------------------

When auditing reports, the shared accounts feature grants a **lighter version of the reports**
while **keeping all the data visible and accessible**. This makes it easier to understand and
manage the accounts.

.. example::
   Auditing amounts in the balance sheet report of a company using a shared account with one of its
   child companies and grouping by account used to show two groups for the two accounts that share
   the same purpose. Now they are shown in the same group and are differentiable using the company
   they are used in.

3. Micro and Macro Management
-----------------------------

For the main companies as much as the last child company of a branch, using the shared accounts
feature allows accountants to **handle the accounts at the level they need to**. This means that
the main accountant can manage the accounts of the whole group while the accountant of a child
company can manage the accounts of that company **without having to worry** about the impact it

> might have on the other companies.

Reviewer: Not sure to get that one :thinking: What do you mean exactly ?  A bit too abstract IMO

Author: I meant to explain that the way the shared accounts is implemented, you have a single account in the database but it is modifiable by each company that shares it. It allows for a managment at both scales, by a main accountant on all companies or by any accountant on the company they handle. Does it make more sense ?

Author: Changed it a bit, tell me what you think :)

.. example::
   Even though the account is shared and is the same record in the database for all companies, its
   code in any company can be modified without affecting the other companies granting a lot of
   flexibility in the management of the accounts.

.. _setting_up_shared_accounts:

Setting Up Shared Accounts
==========================

Enable Multi-Company
--------------------

To enable account sharing, you first need to enable the multi-company feature in Odoo. This can be

> done by simply making sure you have multiple companies setup in your Odoo database.

Reviewer: Do we want to say it ? Do we want to say it this way ? Isn't that obvious ? Also, can't we create confusion with branches by only saying "multi-company" (even though we know branches are just part of multi-company)

Author: I thought it wouldn't hurt to say it since the "Companies" field in the "Accounting" tab only shows up in multi-company. So people might find it weird that we say to use a field that is not shown... I guess both are defensible ? What would you rather do ?

Configure Shared Accounts
-------------------------

Once multi-company is enabled, you can configure shared accounts by going to an account you wish to
share between companies. Head into the :guilabel:`Mapping` tab and enter the code you want for the
account for each desired company.

.. image:: shared_accounts/shared_accounts_configure_shared_accounts_mapping.png
   :alt: Configuring company relative codes for a shared account in Odoo Accounting

Afterwards, you can head back to the :guilabel:`Accounting` tab and add your second company to the
list of companies this account is available for.

.. image:: shared_accounts/shared_accounts_configure_shared_accounts_accounting.png
   :alt: Adding a company to a shared account in Odoo Accounting

.. _merge_tool:

Merge Tool
==========

Use cases
---------

> The merging tool that accompanies the new shared accounts features is there to simplify the process

Reviewer: ```suggestion
The merging tool is there to simplify the process
```

of creating shared accounts. This tool is particularly useful when you have an **already setup
multi-company environment** in which you were using separate accounts that you now want to merge.

Steps
-----

> To use the merge tool of the shared accounts feature, you need to select all the companies that

Reviewer: ```suggestion
To use the merge tool, you need to select all the companies that
```

have an account you want to merge in the company selector in the top right corner of the screen.

.. image:: shared_accounts/shared_accounts_merge_tool_select_companies.png
   :alt: Selecting all companies that have accounts to be merged.

You can then select all the account that are to be merged, click the :guilabel:`Actions` button,
and select the :guilabel:`Merge accounts` option in the drop-down.

.. image:: shared_accounts/shared_accounts_merge_tool_merge_action.png
   :alt: Selecting the accounts to merge and the tool in the options.

> You will then enter a confirmation wizard where you can select the account you want to merge and
> confirm the operation. Once all is set, you can click the :guilabel:`Merge` button to complete the

Reviewer: ```suggestion
You will then enter a confirmation wizard where you can 
confirm the operation. Once all is set, you can click the :guilabel:`Merge` button to complete the
```

In the end, the selected accounts will be merged into a single shared account that is available for
all the selected companies in the same way it would have been if the account had only been created
on a single company and then configured to be shared like in the :ref:`basic setup
<setting_up_shared_accounts>`.

.. image:: shared_accounts/shared_accounts_merge_tool_result.png
   :alt: Result of the Merge Tool of the shared accounts feature.

.. _unmerge_tool:

Unmerge Tool
============

Use cases
---------

At the opposite end of the spectrum, the unmerge tool is there to simplify the process of stopping
to share specific accounts. This tool finds its use when you have shared accounts that you want to
**dissociate from certain companies** without losing the data that they contain.

Steps
-----

Using any of the companies that have a shared account, it is possible to select it and unmerge it
using the :guilabel:`Unmerge account` option in the :guilabel:`Actions` menu.

.. image:: shared_accounts/shared_accounts_unmerge_tool_unmerge_action.png
   :alt: Selecting an account to unmerge and the tool in the options.

> This will open a warning window confirming all the accounts that will be created if the action is

Reviewer: ```suggestion
This will open a confirmation window listing all the accounts that will be created if the action is
```

> This will open a warning window confirming all the accounts that will be created if the action is
> confirmed using for each of them the code setup for the company it will be created in.

Reviewer: ```suggestion
This will open a warning window confirming all the accounts that will be created.
```
Not need to say too much IMO ; that's implicit anyway, and keeping the sentence simpler makes it easier to understand :)

.. image:: shared_accounts/shared_accounts_unmerge_tool_confirmation_wizard.png
   :alt: Confirmation wizard for the Unmerge Tool of the shared accounts feature.

In the end, you will find in each of the companies that had a code configured on the shared

> account a new account using that same code splitting the hared one into separate accounts.

Reviewer: ```suggestion
account a new account using that same code splitting the shared one into separate accounts.
```

> account a new account using that same code splitting the hared one into separate accounts.

Reviewer: Maybe we can make this simpler a little simpler as well :thinking:

Author: I tried something, tell me what you think.
